### PR TITLE
Update sbc-common-components version number

### PR DESCRIPTION
**Pull request by eve-git**
*Issue #:* /bcgov/entity#6682

*Description of changes:*
Update sbc-common-components version number

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "launchdarkly-js-client-sdk": "^2.18.0",
     "lodash": "^4.17.20",
     "regenerator-runtime": "^0.13.7",
-    "sbc-common-components": "2.4.17",
+    "sbc-common-components": "^2.4.18",
```
Comment (severinbeauvais) [inline package.json:27]: Remove ^.

**Review by severinbeauvais — CHANGES_REQUESTED** [review]
~See my comment.~ _now fixed, thanks_

File: package.json (hunk 1 above)
Comment (eve-git) [reply package.json]: ^ removed. The change pushed to origin.

**Review by severinbeauvais — APPROVED** [review]
Looks good. Thanks for the update.

@andrepestana-aot If you approve this PR, please go ahead and merge it.

**Review by andrepestana-aot — APPROVED** [review]
👍

**Pull request merged**